applications/ortege.md

```diff
@@ -0,0 +1,264 @@
+
+- https://github.com/SarjuHansaliya 
+- https://github.com/HariSeldon23 
+- https://github.com/bmino 
+- https://github.com/prodesert22 
+- https://github.com/josedev-union 
+
+### Team LinkedIn Profiles (if available) 
+
+ - https://www.linkedin.com/in/shane-mccann-394a8425/ 
+- https://www.linkedin.com/in/justintrollip/ 
+- https://www.linkedin.com/in/firat-akkan/ 
+
+ ## Development Status :open_book: 
+ Our project, Ortege, has made significant strides in its development phase. We have successfully deployed our backend system in a Kubernetes environment, which includes the operation of validators and relayers. This robust infrastructure ensures the smooth and secure functioning of our protocol. 
+
+In the spirit of decentralization and resilience, we have designed our system to allow for the deployment of additional validators by any party. This feature enhances the security and reliability of our protocol by preventing any single point of failure and promoting a more distributed network. 
+
+We have also made substantial progress on the smart contract front. Our team has deployed smart contracts onto EVM chains and achieved key milestones, including the successful transmission of interchain messages, the creation of a bridge for tokens, and the ability to remotely read state on smart contracts. These accomplishments demonstrate our technical prowess and the functional capabilities of our protocol. 
+
+To provide transparency and traceability, we have deployed an explorer that can track the entire lifecycle of a message. This tool allows users to monitor the status of their transactions and provides valuable insights into the operation of the protocol. 
+
+Lastly, we have integrated WalletConnect 2 to enable the remote control of an interchain account on another chain. This feature enhances the user experience by allowing users to manage their assets across different chains from a single interface. 
+
+In summary, while there is still work to be done, we are proud of the progress we have made so far and are excited about the potential of Ortege to revolutionize blockchain interoperability. 
+
+## Development Roadmap :nut_and_bolt: 
+ ### Overview 
+- **Total Estimated Duration:** 5 months 
+
+- **Full-Time Equivalent (FTE):**  4 FTE (Milestones will be worked on concurrently) 
+
+- **Total Costs:** $90,000 
+
+### Milestone 1 Ink! Smart Contracts 
+
+We will begin by developing smart contracts in the ink! language. These contracts will form the backbone of our protocol, enabling the core functionality of cross-chain communication. We will leverage the design principles and specifications provided by Hyperlane to ensure our contracts are robust, secure, and efficient. This will involve creating contracts for managing the protocol's state, handling message routing, and facilitating the transfer of value across chains. 
+ 
+- **Estimated duration:** 4 months 
+- **FTE:**  2.5 
+- **Costs:** 60,000 USD 
```

Comment:
The milestones still have different per-person rates: 6k for M1, 5k for M2 and M3.
Why is that?